docs/narr/viewconfig.rst

```diff
@@ -506,20 +508,22 @@ configured view.
 
 ``custom_predicates``
   If ``custom_predicates`` is specified, it must be a sequence of references to
-  custom predicate callables.  Use custom predicates when no set of predefined
-  predicates do what you need.  Custom predicates can be combined with
+  custom predicate callables.   Custom predicates can be combined with
   predefined predicates as necessary.  Each custom predicate callable should
   accept two arguments, ``context`` and ``request``, and should return either
   ``True`` or ``False`` after doing arbitrary evaluation of the context
   resource and/or the request.  If all callables return ``True``, the
   associated view callable will be considered viable for a given request.
+  This parameter is kept around for backward compatibility.
 
-  If ``custom_predicates`` is not specified, no custom predicates are used.
+  .. deprecated:: 1.5
+     See section below for new-style custom predicates.
 
-``predicates``
-  Pass a key/value pair here to use a third-party predicate registered via
```

Comment:
I didn’t like `third-party` here, as custom predicates can be written by the dev, who is the first party!

Reply:
OK to change to `custom predicate` in all the spots you noted!

I still don’t get why third-party packages are mentioned.  The app I am writing to test Pyramid 2 has no third-party custom predicates, only mine.